homeassistant/components/renson/services.yaml

```diff
@@ -0,0 +1,216 @@
+set_manual_level:
+  name: Set Manual Level
+  description: Sets manual level
+  fields:
+    manual_level:
+      name: Manual level
+      description: Manual level setting
+      required: true
+      advanced: false
+      default: "Off"
+      selector:
+        select:
+          options:
+            - "Off"
+            - "Level1"
+            - "Level2"
+            - "Level3"
+            - "Level4"
+            - "Breeze"
+            - "Holiday"
+
+sync_time:
+  name: Synchronize time and date
+  description: Synchronize time and date with device
+
+set_timer_level:
+  name: Set timer
+  description: Set the ventilation timer
+  fields:
+    timer_level:
+      name: Level
+      description: Level setting
+      required: true
+      advanced: false
+      default: "Level1"
+      selector:
+        select:
+          options:
+            - "Level1"
+            - "Level2"
+            - "Level3"
+            - "Level4"
+            - "Holiday"
+            - "Breeze"
+    time:
+      name: Time
+      description: Time of the timer (0 will disable the timer)
+      required: true
+      advanced: false
+      default: 0
+      selector:
+        number:
+          min: 0
+          max: 1440
+          step: 10
+          unit_of_measurement: "min"
+          mode: slider
+
+set_breeze:
+  name: Set breeze
+  description: Set the breeze function of the ventilation system
+  fields:
+    breeze_level:
+      name: Level
+      description: Ventilation level when breeze function is activated
+      required: false
+      advanced: false
+      default: "Level3"
+      selector:
+        select:
+          options:
+            - "Level1"
+            - "Level2"
+            - "Level3"
+            - "Level4"
+    temperature:
+      name: Temperature
+      description: Temperature when the breeze function should be activated
+      required: false
+      advanced: false
+      default: 18
+      selector:
+        number:
+          min: 15
+          max: 35
+          step: 1
+          unit_of_measurement: "°C"
+          mode: slider
+    activate:
+      name: Activate
+      description: Activate or disable the breeze feature
+      required: true
+      advanced: false
+      default: false
+      selector:
+        boolean:
+
+set_day_night_time:
```

Comment:
Can be `Time` entities?

Reply:
I dont' think this is possible in a Time entity. The set_day_night_time service is updating two fields at the same time.

Reply:
I remove this and will add the time entities to a new PR

Reply:
Opened https://github.com/home-assistant/core/pull/105031 for replacing the setters